source/Applications/Advanced/HandEyeCalibration/PoseConversions/PoseConversions.cpp

```diff
@@ -66,12 +61,32 @@ int main()
         std::cout << "RotationMatrix:\n" << transformationMatrix.linear() << std::endl;
         std::cout << "TranslationVector:\n" << transformationMatrix.translation() << std::endl;
 
-        // Convert from Zivid to Robot (Transformation Matrix --> any format)
-        const auto robotRotationRepresentations = zividToRobot(transformationMatrix);
-
-        // Convert from Robot to Zivid (any format --> Rotation Matrix)
-        const auto transformationMatrix2 =
-            robotToZivid(robotRotationRepresentations, transformationMatrix.translation());
+        /*
+         * Convert from Zivid to Robot (Transformation Matrix --> any format)
+         */
+        print_header("Convert from Zivid (Rotation Matrix) to Robot");
+        const Eigen::AngleAxisd axisAngle(transformationMatrix.linear());
```

Comment:
I am not sure if this makes sense but it would look nicer. @knatten please comment.

Create a function called rotationMatrixToAngleAxis which just executes this single line. The reason is to have all functions with similar names:
rotationMatrixToAngleAxis
rotationMatrixToRoatationVector
rotationMatrixToQuaternion
rotationMatrixToListOfRollPitchYaw

Reply:
I thought about it, but didn't do it since it's already just one line. It would add 5 lines minimum to add a function wrapper. @knatten?

Reply:
No strong opinion on this, I think both ways are ok.

Reply:
Your call then @torbsorb